radio/src/gui/colorlcd/radio_hardware.cpp

```diff
@@ -287,7 +296,16 @@ void RadioHardwarePage::build(FormWindow * window)
 
 #if defined(AUX2_SERIAL)
   new StaticText(window, grid.getLabelSlot(), STR_AUX2_SERIAL_MODE);
+//OW
+#if defined(TELEMETRY_MAVLINK) && !defined(CLI) && !defined(DEBUG)
```

Comment:
Here again, whereby we have 2 AUX serial ports, activating mavlink prevents debugging via serial port, whereby it could be still enabled.